plugins/aws_ec2/help.md

```diff
@@ -14,17 +14,31 @@ to general management of EC2 instances.
 * An AWS account with admin permissions for EC2
 * An AWS Access Key ID and Secret Access Key
 
+# Supported Product Versions
+
+_There are no supported product versions listed._
+
```

Comment:
Kindly please `make regenerate` to fit `supported_versions` field form plugin.spec.yaml file 😃